### DISCUSSION TITLE
Custom URL for inserted images in WYSIWYG editor

### DISCUSSION THREAD

**Discussion by tinemlakar** [directus/directus#5371] [Archive]
I would like to be able to get direct link to my public S3 storage (or custom URL) location of the image inserted in WYSIWYG editor. Is that possible to achieve with some settings or is this a feature request? :)

Thanks for the help.

**Comment by rijkvanzanten** [comment]
That's not currently possible 👍🏻

**Reply by tinemlakar** [reply]
OK. Thank you for quick answer.
